src/services/sqs-service.ts

```diff
@@ -34,9 +34,7 @@ class SqsService {
           QueueUrl: this.sqsQueueURL,
         }),
       )
-
       console.log('Success, message sent. MessageID:', data.MessageId)
-
```

Comment:
I don't believe you need to log anything there. You have a log outside of this function that has the proper tags. Please remove.